content/slice/language-guide/parameters.md

```diff
@@ -76,11 +81,12 @@ case. The type of the C# field is the mapped C# type for `Type`.
 Tagged parameters are mapped just like regular parameters. The tag and tag number don't appear in the mapped C# API.
 
 {% slice2 %}
-### Stream parameters in C#
+### Stream parameters in C #
```

Comment:
This doesn't look correct, the linter seems to get confused when a title ends with `#`